deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -79,18 +78,15 @@ if [ "${DISCONNECTED}" = "true" ]; then
     mkdir -p ./hypershift-cli
     podman cp $id:/usr/bin/hypershift ./hypershift-cli
     export PATH="$PATH":"$PWD"/hypershift-cli
-    # 4. mirrored openshift release to local registry
-    # disconnected openshift release image will be used as release-image flag for hypershift create cluster
-    oc image mirror -a "${PULL_SECRET_FILE}" "${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE}" "${DISCONNECTED_ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE}"
-    export ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE="${DISCONNECTED_ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE}"
-    # 5. mirrored capi agent image to local registry
+    # 4. mirrored capi agent image to local registry
     if [ ! -z "$PROVIDER_IMAGE" ]
     then
       export PROVIDER_LOCAL_IMAGE="${LOCAL_REGISTRY}/localimages/cluster-api-provider-agent:latest"
       oc image mirror -a "${PULL_SECRET_FILE}" "${PROVIDER_IMAGE}" "${PROVIDER_LOCAL_IMAGE}"
       export PROVIDER_IMAGE="${PROVIDER_LOCAL_IMAGE}"
     fi
-    # 6. ImageContentPolicy for local registry
+    # 5. ImageContentPolicy for local mirror registry (prerequisite is the openshift release is mirrored to the local registry)
+    export OCP_MIRROR_REGISTRY="${LOCAL_REGISTRY}/$(get_image_repository_only ${ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE})"
```

Comment:
OCP release is mirrored here
https://github.com/openshift/assisted-service/blob/571e57d934395c1d23c52048b9e31d701a50cd6a/deploy/operator/deploy.sh#L40